vllm/v1/core/scheduler_output.py

```diff
@@ -111,3 +114,9 @@ class SchedulerOutput:
     # list of (req_id, encoder_input_index) tuples.
     # Used to free the encoder cache.
     free_encoder_input_ids: list[tuple[str, int]]
+
+    # Dict of request ids to their index within the batch
+    # for filling the next token bitmask
+    structured_output_request_ids: dict[str, int]
+    # the bitmask for the whole batch
+    grammar_bitmask: Optional["npt.NDArray[np.int32]"]
```

Comment:
Just curious: Can't we just do `np.ndarray`?

Reply:
oh, this is because mypy will throw an error of `np.ndarray[unknown, unknown]`. We can also write `np.ndarray[Any, np.dtype[np.int32]]`, but it is equivalent to `npt.NDArray[np.int32]`